### ISSUE TITLE
How to build a model layers

### ISSUE THREAD

**Issue by ior308** [nlpodyssey/cybertron#36]
Dear,

  I'm looking for how to write the following model built in Keras using cybertron or spago:

model = tf.keras.models.Sequential()

# Input - Layer\n",
model.add(layers.Dense(8, activation=\"relu\", input_shape=(10000, )))

# Hidden - Layers
model.add(layers.Dense(4, activation=\"relu\"))
model.add(layers.Dense(2, activation=\"relu\"))

# Output- Layer\n",
model.add(layers.Dense(1, activation=\"sigmoid\"))

# report architettura modello
model.summary()

Could you give me an example ? Obviously it's a simple example, but if it were possible, as with Keras, complex architectures can be built, even CCN and RNN or Seq2Seq.

Regards
